spec/factories/attachments.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
```

Comment:
why remove this? you're still going to use the attachment factory in the attachable attachment factory?